catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/query/cql/CqlRequest.java

```diff
@@ -141,6 +156,12 @@ public QueryRequest createQueryRequest(String localSource, FilterBuilder filterB
           .put("excludeAttributes", Sets.newHashSet(Metacard.METADATA, "lux"));
     }
 
+    if (sortBys.size() > 1) {
+      queryRequest
+          .getProperties()
+          .put("additional.sorts.bys", sortBys.subList(1, sortBys.size()).toArray(new SortBy[0]));
```

Comment:
is there any reasonable way (and there may not be) to make all the places that use this value leverage a single constant?

Reply:
I asked myself the same thing when I was writing this. All the declarations I could find were private. Is it worth moving this to `Constants` or somewhere else? I am not at all opposed to doing so but I was not able to determine the reason for the current state of affairs either. https://github.com/codice/ddf/search?utf8=%E2%9C%93&q=additional.sort.bys&type=

Reply:
This has been refactored now.